task/06-conditions-n-loops-tasks.js

```diff
@@ -251,7 +284,14 @@ function reverseInteger(num) {
  *   4916123456789012 => false
  */
 function isCreditCardNumber(ccn) {
-    throw new Error('Not implemented');
+    let reverted = ccn.toString().split('').reverse();
+    let s1 = reverted.reduce((acc, v, i)=>
+        i % 2 == 0 ? acc + +v : acc, 0);
+    let s2 = reverted.reduce((acc, v, i) =>
+         i % 2 != 0 
```

Comment:
"operator !== is always preferable
sometimes ''!=' may cause an error
(check at console :
""0"" != 0
and ""0"" !== 0)"